### ISSUE TITLE
translate Errorcode into Text

### ISSUE THREAD

**Issue by sokatra** [OCSInventory-NG/OCSInventory-ocsreports#784]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
It would be fine if OCS-Inventory is able to translate error codes into Text in a second column
For example i create some scripts with the following windows cmd command
exit /B 2000

The errorcode reported to ocs is 2000 and i would like to have a translation into a text. For example 

error 2000 means - the Antivirus Service is running
because my script is checking for the running service and doesn't install something if the service isn't running.
Sometimes i have 5 different errorcodes which are reported back to ocs-inventory so i know what is going on on the client.
There must be a predefined errocode LISTwith standard error codes like 1603 which is edititable and can be extended.
I know what errocode 2000 means - but my colleagues didn't 
In short - make errorcodes human readable
**Describe alternatives you've considered**

**Additional context**

**Comment by gillesdubois** [comment]
HI @sokatra ,

Unfortunatly, OCS rely on system return code by default and there is no plan to change that.
In the case you want to use you proper exit code, you will have to document them so your team can know what each code mean.

Regards,
Gilles Dubois.

**Comment by sokatra** [comment]
ok thanks. but perhaps i haven't described my suggestion sufficient. so please have a look at the professional deploy software PQD Deploy
Step 8
https://adminarsenal.zendesk.com/hc/en-us/articles/220509287-How-It-Works-PDQ-Deploy

so OCS-Inventory can furthermore rely on errorcodes , but only with the extension that the codes are human readable. I think this can be a major step for the usability and because of its simplicity it should be easy to impement into ocs-inventory web console - without changing anything on the client.
I don't want to have that ocs-inventory deployment system has to be changed in this case(relying on errocodes).
i only think it would be a profitable for the whole great ocs-inventory project if users (administrators) have nice interface to get their work done.
Kind regards
sokatra
